view/frontend/web/js/view/payment/method-renderer/billink.js

```diff
@@ -263,7 +263,7 @@ define(
                                     max: 12
                                 },
                                 'BE': {
-                                    min: 10,
+                                    min: 9,
```

Comment:
I thought it was already fix today

Reply:
it was AP - https://github.com/buckaroo-it/Magento2/pull/293/commits/a25ad4abfef25ab285b3d0509bb9bd2fb83bd71a

Reply:
sorry, thx